java/src/main/java/com/cybersource/ws/client/SecurityUtil.java

```diff
@@ -330,7 +330,7 @@ private static void loadJavaKeystore(MerchantConfig merchantConfig, Logger logge
 		try {
 			is = new FileInputStream(merchantConfig.getKeyFile());
```

Comment:
here as well.

Reply:
Its already been closed in the finally block.So , I dint make any changes.